src/components/modal/README.md

```diff
@@ -471,6 +472,35 @@ To disable stacking for a specific modal, just set the prop `no-stacking` on the
 - The opaque backdrop will appear progressively darker for each modal that is opened. This is expected behaviour as each backdrop is opened over top the other backdrops.
 - For multiple modals to stack properly, they **must** be defined in the document in the order they will be opened, otherwise a newly opened modal may appear hidden or obscured by a currently open modal.
 
+
+## Listen for a modal changes via $root events
+
+To listen for opening any modals, use:
+
+```js
+mounted () {
+  this.$root.$on("bv::show::modal", function(modalId, elIDtoFocusOnClose) { {
+    console.log("modalId:", modalId);
+    console.log("elIDtoFocusOnClose:", elIDtoFocusOnClose);
+  });
+}
+```
+
+where `modalId` is modal id to be shown; `elIDtoFocusOnClose` is id of element to return focus to once the modal is closed.
+
+To listen for hiding any modals, use:
```

Comment:
Again, as above this is misleading as it is **not** the modal that emits this event, but rather `v-b-toggle` (or user code that wants to programmatically hode a modal).

As above, this is a control event, not a status event!